node-tests/.eslintrc.js

```diff
@@ -2,7 +2,11 @@
 
 module.exports = {
   root: true,
-  extends: 'eslint:recommended',
+  extends: ['eslint:recommended'],
+  parserOptions: {
+    ecmaVersion: 2017,
+    sourceType: 'module'
```

Comment:
that seems wrong to me. are we using ES6 modules for the Node.js-based tests?

Reply:
You are correct. Will try to remove.